src/chronos/chronos.py

```diff
@@ -538,6 +536,52 @@ def predict(
 
         return torch.cat(predictions, dim=-1)
 
+    def predict_quantiles(
+        self,
+        context: Union[torch.Tensor, List[torch.Tensor]],
+        prediction_length: Optional[int] = None,
+        quantile_levels: List[float] = [0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9],
+        **predict_kwargs,
+    ) -> Tuple[torch.Tensor, torch.Tensor]:
+        """
+        Get quantile and mean forecasts for given time series.
+
+        Parameters
+        ----------
+        context : Union[torch.Tensor, List[torch.Tensor]]
+            Input series. This is either a 1D tensor, or a list
+            of 1D tensors, or a 2D tensor whose first dimension
+            is batch. In the latter case, use left-padding with
+            ``torch.nan`` to align series of different lengths.
+        prediction_length : Optional[int], optional
+            Time steps to predict. Defaults to a model-dependent
+            value if not given.
+        quantile_levels : List[float], optional
+            Quantile levels to compute, by default [0.1, 0.2, ..., 0.9]
+
+        Returns
+        -------
+        quantiles
+            Tensor containing quantile forecasts.
+            Shape: (batch_size, prediction_length, num_quantiles)
+        mean
+            Tensor containing mean (point) forecasts.
+            Shape: (batch_size, prediction_length)
+        """
```

Comment:
I see there are a lot of repeated docstrings (not just this class/method) should we find a way to make it less redundant? Not a lot of maintenance going on, but I'm wondering whether referencing the docstring of the base class and focus on the differences (added args) would be better

Reply:
Added reference to the base class.